data/src/main/java/io/gnosis/data/backend/dto/TransactionDto.kt

```diff
@@ -30,7 +31,14 @@ sealed class ParamDto {
         override val type: String,
         override val name: String,
         override val value: List<Any>
-    ) : ParamDto()
+    ) : ParamDto() {
+
+        fun getItemType(): ParamType = when (type.split("[")[0]) {
```

Comment:
Just to clarify: This could me more than one type. ATM we only look at the first to determine the array param type?

Reply:
we just need to know if it's an array